src/appendFlipped.js

```diff
@@ -0,0 +1,25 @@
+import { append, flip } from 'ramda';
+
+
+/**
+ * Returns a new list containing the contents of the given list, followed by the given element.
+ * Ramda's `append`, but with argument order reversed: list then value.
```

Comment:
```js
Returns a new list containing the contents of the given list, followed by the given element.
Like {@link http://ramdajs.com/docs/#append|R.append} but with arguments order reversed.
```
Would drop the `list then value` part. It's apprarent from the signature and jsdoc @params.